install/set_gigabyte_node_bmc_to_factory_defaults.md

```diff
@@ -1,111 +1,106 @@
 # Set Gigabyte Node BMC to Factory Defaults
 
-### Prerequisites
+## Prerequisites
+
+**Note**: This section refers to scripts that exist only in the PIT environment. If necessary, copy the LiveCD data from a different machine to get these scripts.
 
 Use the management scripts and text files to reset Gigabyte BMC to factory default settings. Set the BMC to the factory default settings in the following cases:
 
-- There are problems using the ipmitool command and Redfish does not respond
-- There are problems using the ipmitool command and Redfish is running
-- When BIOS or BMC flash procedures fail using Redfish
+- There are problems using the `ipmitool` command and Redfish does not respond
+- There are problems using the `ipmitool` command and Redfish is running
+- When BIOS or BMC flash procedures fails using Redfish
   - Run the `do_bmc_factory_default.sh` script
-  - Run `ipmitool -I lanplus -U admin -P password -H BMC_or_CMC_IP mc reset cold` and flash it again after 5 minutes seconds
+  - Run `ipmitool -I lanplus -U admin -P password -H BMC_or_CMC_IP mc reset cold` and flash it again after 5 minutes (300 seconds).
 - If booted from the PIT node:
-  - the firmware packages are located in the HFP package provided with the Shasta release
-  - the required scripts are located in `/var/www/fw/river/sh-svr-scripts`
+  - The firmware packages are located in the HFP package provided with the Shasta release
+  - The required scripts are located in `/var/www/fw/river/sh-svr-scripts`
 
-### Procedure
+## Procedure
 
-#### Apply the BMC Factory Command
+### Apply the BMC Factory Command
 
 1. Create a `node.txt` file and add the target node information as shown:
 
-   Example `node.txt` file with two nodes:
+    Example `node.txt` file with two nodes:
 
-   ```screen
+    ```screen
```

Comment:
I've seen people using it, but I don't think "screen" is one of the standard values used for markdown fenced code blocks., At least [not according to this file](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) (which is what GitHub appears to consider standard). I think for these examples, "text" makes sense.
```suggestion
    ```text
```